crates/nu-std/std/math.nu

```diff
@@ -0,0 +1,22 @@
+# return the Euler-Mascheroni constant
+#
+# see https://en.wikipedia.org/wiki/Euler%27s_constant
+export def gamma [] { 0.5772156649015329 }
```

Comment:
feels like these should be `export const` but not sure if we support that yet

Reply:
very good catch, i think we do now :open_mouth:

Reply:
done in 57da75580

Reply:
i've updated the description of the PR with the possible usage :+1: 

> **Note**
> `use std math; $math` does not work, but i think it should :thinking: